updating/understanding-openshift-updates.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Understanding {product-title} updates
+include::modules/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update a {product-title} cluster with a single click using over the air update.
```

Comment:
I'm not a big fan of "over the air", because things work pretty much the same way in restricted networks (although it may take a few setup steps to get the target bits somewhere the cluster can reach them).  Maybe "... with a single operation." or some such?